hardware_interface:
    loop_hz: 10 # hz
    joints:
        - j1
        - j2
        - j3
        - j4
        - j5
        - j6
